stream-chat-android-state/src/main/java/io/getstream/chat/android/offline/event/handler/internal/EventHandlerSequential.kt

```diff
@@ -451,13 +452,16 @@ internal class EventHandlerSequential(
                     event.message.enrichWithCid(event.cid)
                     event.message.enrichWithOwnReactions(batch, currentUserId, event.user)
                     batch.addMessageData(event.cid, event.message, isNewMessage = true)
-                    repos.selectChannelWithoutMessages(event.cid)?.let { channel ->
-                        val updatedChannel = channel.copy(
-                            hidden = false,
-                            messages = listOf(event.message)
-                        )
-                        batch.addChannel(updatedChannel)
-                    }
+                    // first check channel in batch, if not found, check in db
+                    //  if we ignore batch, we may override existing channel in batch with the one from db
+                    val channel = batch.getCurrentChannel(event.cid)
+                        ?: repos.selectChannelWithoutMessages(event.cid)
+                        ?: continue
+                    val updatedChannel = channel.copy(
+                        hidden = false,
+                        messages = listOf(event.message)
+                    )
+                    batch.addChannel(updatedChannel)
```

Comment:
### Fix 2
We get `channel` from batch and if not found fetch it from DB.

Explanation.
When user is being deleted from the channel we receive up to 6 events (applied from top to bottom):
- `notification.removed_from_channel`
- `notification.mark_read`
- `member.removed`
- `message.new`
- `channel.updated`
- `channel.kicked`

So if fetch immediately from DB we will override all applied changes from events.